geometry/symmetric_bilinear_form.hpp

```diff
@@ -64,6 +64,10 @@ class SymmetricBilinearForm {
            typename = std::enable_if_t<base::is_same_template_v<M, Bivector>>>
   SymmetricBilinearForm<Scalar, Frame, Vector> AnticommutatorInverse() const;
 
+  template<typename S = Scalar,
+           typename = std::enable_if_t<std::is_same_v<S, double>>>
+  static SymmetricBilinearForm<S, Frame, Multivector> Identity();
```

Comment:
This is `InnerProductForm`; remove this misnomer (this is not an endomorphism) and use `InnerProductForm`.

Reply:
Done.